services/matrix/matrix.tester.js

```diff
@@ -439,7 +521,36 @@ t.create('specify the homeserver fqdn')
     color: 'brightgreen',
   })
 
-t.create('test on real matrix room for API compliance')
+t.create('specify the homeserver fqdn for summary')
+  .get('/ALIAS:DUMMY.dumb.json?server_fqdn=matrix.DUMMY.dumb&fetchMode=summary')
+  .intercept(nock =>
+    nock('https://matrix.DUMMY.dumb/')
+      .get(
+        '/_matrix/client/unstable/im.nheko.summary/rooms/%23ALIAS%3ADUMMY.dumb/summary',
+      )
+      .reply(
+        200,
+        JSON.stringify({
+          num_joined_members: 4,
+        }),
+      ),
+  )
+  .expectBadge({
+    label: 'chat',
+    message: '4 users',
+    color: 'brightgreen',
+  })
+
+t.create('test on real matrix room for guest API compliance')
+  .get('/ndcube:openastronomy.org.json?server_fqdn=openastronomy.modular.im')
+  .timeout(10000)
+  .expectBadge({
+    label: 'chat',
+    message: Joi.string().regex(/^[0-9]+ users$/),
+    color: 'brightgreen',
+  })
+
+t.create('test on real matrix room for summary API compliance')
```

Comment:
There's a couple of tests here with duplicated names. Could you given them unique names

Reply:
I don't see duplicates, what exactly do you mean?

Reply:
Sorry. This is me failing to read the diff correctly :facepalm:  Ignore this